ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -1676,12 +1691,35 @@ - (BOOL)isEqual:(id)object {
              self.autocapitalizationType == castObject.autocapitalizationType &&
              self.autocorrectionType == castObject.autocorrectionType &&
              self.spellCheckingType == castObject.spellCheckingType &&
+             self.keyboardType == castObject.keyboardType &&
              self.multipleLines == castObject.multipleLines));
 }
 
 @end
 
 
+#pragma mark - ORKEmailAnswerFormat
+
+@implementation ORKEmailAnswerFormat
+
+- (Class)questionResultClass {
+    return [ORKTextQuestionResult class];
+}
+
+- (ORKAnswerFormat *) impliedAnswerFormat{
```

Comment:
Remove space before `impliedAnswerFormat`, and add a space between `impliedAnswerFormat` and `{`.